.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -575,14 +575,14 @@ stages:
       displayName: Download MSI
       inputs:
         artifact: windows-msi-$(targetPlatform)
-        patterns: '**/*profiler-beta.msi'
+        patterns: '**/*.msi'
```

Comment:
Note that previously this matched a _single_ `.msi` file, and now it matches two, the x64 and x86 msi. I think we should _either_

- Restrict this to the x64 installer, e.g. `'**/*-x64.msi'`
- Use the platform-specific installer, e.g. `'**/*$(targetPlatform).msi'`

My preference is for the former, as I think it's what we were doing before, it's what most people will use (even when running in 32 bit mode), and the yaml will definitely work 🙂

Reply:
Taking the currently in the `windows-msi-x64` artifact as an example, we currently have:
- `datadog-dotnet-apm-2.6.0-x64-profiler-beta.msi`
- `datadog-dotnet-apm-2.6.0-x64.msi`

After the changes in this PR, only the `datadog-dotnet-apm-2.6.0-x64.msi` should remain so I think this is fine. I would feel more comfortable if this PR also removed the `BuildMsi` step from Nuke and the pipeline steps so only one step is responsible for building the installer. As the changes currently stand, both the `BuildMsi` and `BuildMsiBeta` both build the same output file so the last one to run overwrites the contents of the file.

Reply:
Thanks for your inputs. I've fixed both in 6dc6dda249c28e100dbe28a2f5ad536fe359d625.
That said, I've also added a new commit 9d28d5a3e44099f39be920011f1023e5590cf895 where I remove completely the tracer (I should have separated all that from the release notes from the beginning). I'm not entirely sure if what I'm doing is correct, especially updating the `UpdateMsiContents`target but it felt right. 
(Note: I haven't tested yet as I was on my Mac, I'll test the targets on my windows laptop now)

Reply:
These changes look good. One way to test the `UpdateMsiContents` target is to delete the Tracer's `.wxs` files and run the target. To ensure there's no diff, you can remove the condition `WIX_IS_NETFRAMEWORK_461_OR_LATER_INSTALLED` which we determined was no longer necessary and you'll need to update the `ComponentGroup` appropriately. Once those are synced we should be good to go

Reply:
Ok I think we're good now